src/network/server.rs

```diff
@@ -404,4 +408,11 @@ impl<N: Network, E: Environment> Server<N, E> {
         // Wait until the heartbeat task is ready.
         let _ = handler.await;
     }
+
+    #[cfg(feature = "prometheus")]
+    fn initialize_metrics(tasks: &mut Tasks<task::JoinHandle<()>>) {
+        tasks.append(snarkos_metrics::initialize().expect("couldn't initialise the metrics"));
+
+        // TODO: set the block height here, as it could be non-zero.
```

Comment:
```suggestion
```

Reply:
Well, we do actually want to set the block height there no?

Reply:
The same principle applies as with my suggestion to remove the (currently) unused `const`s - it doesn't apply yet.